tests/nix_tests/test_ics20_precompile.py

```diff
@@ -1001,10 +1004,11 @@ def check_contract_balance():
 
     final_dest_balance = 0
 
+    # TODO: this is unused - can we remove it or should we check it somewhere?
     def check_dest_balance():
         nonlocal final_dest_balance
         final_dest_balance = get_balance(
-            ibc.chains["chainmain"], dst_addr, EVMOS_IBC_DENOM
+            ibc.chains["chainmain"], dst_address, EVMOS_IBC_DENOM
         )
         receipt = send_transaction(ibc.chains["evmos"].w3, approve_tx, KEYS["signer2"])
         assert receipt.status == 1, f"Failed: {name}"
```

Comment:
@GAtom22 this whole function is not used - can we remove it or has there been some test logic accidentally removed that required this?